src/routes/safe/components/Transactions/TxsTable/columns.js

```diff
@@ -46,9 +55,22 @@ export const getTxAmount = (tx: Transaction) => {
 
 export type TransactionRow = SortRow<TxData>
 
+export const getIncomingTxTableData = (incomingTransactions: List<IncomingTransaction>): List<TransactionRow> => {
+  return incomingTransactions.map((tx: IncomingTransaction) => ({
+    [TX_TABLE_ID]: tx.blockNumber,
```

Comment:
I don't think this will play well with pending outgoing transactions, probably something like `tx.blockNumber || tx.nonce` would fix the issue

Reply:
But we don't have `nonce` for the incoming txs

Reply:
Yeah, right. But the thing is that pending transactions don't have the `blockNumber` 😄 how did you solve that?

Reply:
actually, thinking on it. I ended up using the date to sort/arrange the elements. Said so, `blockNumber`/`nonce` are unnecessary for this table :thinking: